server/request_handler.go

```diff
@@ -59,6 +60,13 @@ func (r *RpcRequestHandler) process() {
 		return
 	}
 
+	var preferences *types.Preferences
```

Comment:
wouldn't even need to be a pointer, could instantiate here already and have every private tx have preferences. wdyt? 🤔